packages/synapse-react-client/src/components/SynapsePriceFeatureRow.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, Typography } from '@mui/material'
+import React, { useMemo } from 'react'
+import { SynapsePriceTableCell } from './SynapsePriceTableCell'
+import { CheckTwoTone } from '@mui/icons-material'
+
+export type SynapsePriceFeatureRowProps = {
+  featureDescription: string
+  standardPlanValue?: boolean | string
+  managedPlanValue?: boolean | string
+  dataCoordinationPlanValue?: boolean | string
+}
+
+export function getPriceFeatureCellContent(value?: boolean | string) {
+  if (value === undefined) {
+    return <></>
+  }
+  if ('boolean' === typeof value) {
+    if (value) {
+      return (
+        <Typography variant="body2">
+          <CheckTwoTone htmlColor="#878E95" />
+        </Typography>
+      )
+    } else {
+      return <></>
+    }
+  }
+  return (
+    <Typography
+      variant="body2"
+      sx={{ marginLeft: '10px', marginRight: '10px' }}
+    >
+      {value}
+    </Typography>
+  )
+}
+
+export const SynapsePriceFeatureRow: React.FunctionComponent<
+  SynapsePriceFeatureRowProps
+> = ({
+  featureDescription,
+  standardPlanValue,
+  managedPlanValue,
+  dataCoordinationPlanValue,
+}) => {
+  const standardPlanElement = useMemo(
+    () => getPriceFeatureCellContent(standardPlanValue),
+    [standardPlanValue],
+  )
```

Comment:
Use JSX in line with above recommendation

Since the value is always a primitive (string/boolean/undefined), I don't think you need to `useMemo` here

```suggestion
  const standardPlanElement = <PriceFeatureCellContent value={standardPlanValue} />
```